src/main/java/frc/robot/subsystems/arm/ArmIOCTRE.java

```diff
@@ -0,0 +1,131 @@
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+
+package frc.robot.subsystems.arm;
+
+import static edu.wpi.first.units.Units.Volts;
+
+import com.ctre.phoenix6.BaseStatusSignal;
+import com.ctre.phoenix6.StatusSignal;
+import com.ctre.phoenix6.configs.TalonFXConfiguration;
+import com.ctre.phoenix6.controls.Follower;
+import com.ctre.phoenix6.controls.PositionVoltage;
+import com.ctre.phoenix6.controls.VoltageOut;
+import com.ctre.phoenix6.hardware.CANcoder;
+import com.ctre.phoenix6.hardware.TalonFX;
+import com.ctre.phoenix6.signals.FeedbackSensorSourceValue;
+import com.ctre.phoenix6.signals.NeutralModeValue;
+import edu.wpi.first.math.filter.Debouncer;
+import edu.wpi.first.units.measure.Angle;
+import edu.wpi.first.units.measure.AngularVelocity;
+import edu.wpi.first.units.measure.Current;
+import edu.wpi.first.units.measure.Voltage;
+
+public class ArmIOCTRE implements ArmIO {
+  public static final double GEAR_RATIO = 1.5;
+
+  public final TalonFX leader = new TalonFX(20);
+  public final TalonFX follower = new TalonFX(21);
+
+  public final CANcoder leaderEncoder = new CANcoder(22);
+
+  private final VoltageOut m_voltReq = new VoltageOut(0.0);
+
+  private final StatusSignal<Angle> leaderPosition = leader.getPosition();
+  private final StatusSignal<AngularVelocity> leaderVelocity = leader.getVelocity();
+  private final StatusSignal<Voltage> leaderAppliedVolts = leader.getMotorVoltage();
+  private final StatusSignal<Current> leaderStatorCurrent = leader.getStatorCurrent();
+  private final StatusSignal<Current> followerStatorCurrent = follower.getStatorCurrent();
+  private final StatusSignal<Current> leaderSupplyCurrent = leader.getSupplyCurrent();
+  private final StatusSignal<Current> followerSupplyCurrent = follower.getSupplyCurrent();
+  private final StatusSignal<Angle> encoderPosition = leaderEncoder.getPosition();
+  private final StatusSignal<AngularVelocity> encoderVelocity = leaderEncoder.getVelocity();
+
+  private final Debouncer leaderDebounce = new Debouncer(0.5);
+  private final Debouncer followerDebounce = new Debouncer(0.5);
+  private final Debouncer encoderDebounce = new Debouncer(0.5);
+
+  public ArmIOCTRE() {
+    var config = new TalonFXConfiguration();
+    config.CurrentLimits.StatorCurrentLimit = 30.0;
+    config.CurrentLimits.StatorCurrentLimitEnable = true;
+    config.MotorOutput.NeutralMode = NeutralModeValue.Coast;
+    config.Feedback.FeedbackRemoteSensorID = leaderEncoder.getDeviceID();
+    config.Feedback.FeedbackSensorSource = FeedbackSensorSourceValue.RemoteCANcoder;
+    config.Slot0.kP = 5;
+    config.Slot0.kI = 0;
+    config.Slot0.kD = 0;
+    leader.getConfigurator().apply(config);
+    follower.getConfigurator().apply(config);
+    follower.setControl(new Follower(leader.getDeviceID(), false));
+
+    BaseStatusSignal.setUpdateFrequencyForAll(
```

Comment:
what does this do? what does 50 mean? why are we setting update frequencies independently on subsystems? (i assume it's for congestion purposes on the bus?)